src/test/java/com/projetoGerenciamentoCurso/GerenciamentoCurso/Controller/AlunoControllerTest.java

```diff
@@ -13,14 +15,15 @@
 
 @SpringBootTest
 @AutoConfigureMockMvc
+@FixMethodOrder(MethodSorters.JVM)

```

Comment:
Adicionar a anotação `@Transactional` em todas as classes de teste, como foi pedido na última revisão
```suggestion
@FixMethodOrder(MethodSorters.JVM)
@Transactional
```

Reply:
feito